sorc/build_gfs_util.sh

```diff
@@ -14,6 +14,12 @@ if [ ! -d "./mkgfsawps.fd" ]; then
    exit
 fi 
 
+if [ $target = jet ]; then
```

Comment:
@lgannoaa 
Instead of spreading the `if [ $target = jet ]; then` blocks across numerous build scripts, please add the following block on [line 42 of build_all.sh](https://github.com/NOAA-EMC/global-workflow/blob/8c777ad04bce0555bed9f6666b808515b0a3766d/sorc/build_all.sh#L42)

```
if [ $target = jet ]; then
  Build_gldas=false
  Build_gfs_util=false
  Build_ww3_prepost=false
fi
```

When eventually we enable Jet for these components, the changes are localized to this file and one does not have to go searching where to enable them.

Reply:
Excellent suggestion, I modified it and removed the similar code from the individual scripts.